cuda_bindings/cuda/bindings/_lib/utils.pyx.in

```diff
@@ -1,662 +1,6 @@
 # SPDX-FileCopyrightText: Copyright (c) 2021-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
 
-from cpython.buffer cimport PyObject_CheckBuffer, PyObject_GetBuffer, PyBuffer_Release, PyBUF_SIMPLE, PyBUF_ANY_CONTIGUOUS
-from libc.stdlib cimport calloc, free
-from libc.stdint cimport int32_t, uint32_t, int64_t, uint64_t
-from libc.stddef cimport wchar_t
-from libc.string cimport memcpy
-from enum import Enum
-from typing import List, Tuple
-import ctypes
-cimport cuda.bindings.cydriver as cydriver
-import cuda.bindings.driver as driver
-cimport cuda.bindings._lib.param_packer as param_packer
-
 ctypedef unsigned long long void_ptr
```

Comment:
Why did we need to keep this `ctypedef`?

Reply:
`utils.pxi` uses `void_ptr` in a few places.  The other places where `utils.pxi` is included define it as well from generated code (which means we can't /also/ define it in `utils.pxi`), and it seemed weird to change the generator to generate all the basic POD types except `void_ptr`.

But, all that said, this entire file (`utils.pyx.in`) should be removed since (I think) we shouldn't be building it at all anymore.